Tests/TestCommon/Mocks/IDXClientAPIMock.swift

```diff
@@ -91,75 +89,75 @@ class IDXClientAPIMock: MockBase, IDXClientAPI {
-                                            "options": options as Any
-                                          ]))
-        completion(result(for: #function))
-    }
-    
-    func resume(completion: @escaping (Result<Response, IDXClientError>) -> Void) {
-        recordedCalls.append(RecordedCall(function: #function, arguments: nil))
-        completion(result(for: #function))
-    }
-    
-    func proceed(remediation option: Remediation, completion: @escaping (Result<Response, IDXClientError>) -> Void) {
-        recordedCalls.append(RecordedCall(function: #function,
-                                          arguments: [
-                                            "remediation": option as Any,
-                                          ]))
-        completion(result(for: #function))
-    }
-    
-    func redirectResult(for url: URL) -> IDXClient.RedirectResult {
-        recordedCalls.append(RecordedCall(function: #function,
-                                          arguments: [
-                                            "url": url as Any
-                                          ]))
-        let result = response(for: #function)
-        return result?["result"] as? IDXClient.RedirectResult ?? .invalidContext
-    }
-    
-    func exchangeCode(redirect url: URL, completion: @escaping (Result<Token, IDXClientError>) -> Void) {
-        recordedCalls.append(RecordedCall(function: #function,
-                                          arguments: [
-                                            "redirect": url as Any
-                                          ]))
-        completion(result(for: #function))
-    }
-
-    func exchangeCode(using remediation: Remediation, completion: @escaping (Result<Token, IDXClientError>) -> Void) {
-        recordedCalls.append(RecordedCall(function: #function,
-                                          arguments: [
-                                            "using": remediation as Any
-                                          ]))
-        completion(result(for: #function))
-    }
-
-    func revoke(token: String, type: String, completion: @escaping (Result<Void, IDXClientError>) -> Void) {
-        recordedCalls.append(RecordedCall(function: #function,
-                                          arguments: [
-                                            "token": token as Any,
-                                            "type": type as Any
-                                          ]))
-        completion(result(for: #function))
-    }
-    
-    func refresh(token: Token, completion: @escaping (Result<Token, IDXClientError>) -> Void) {
-        recordedCalls.append(RecordedCall(function: #function,
-                                          arguments: [
-                                            "token": token as Any
-                                          ]))
-        completion(result(for: #function))
-    }
-}
+//class IDXClientAPIv1Mock: MockBase, IDXClientAPIImpl {
```

Comment:
Fix or remove?

Reply:
Thanks for catching that; I thought I'd gotten them all.